src/sentry/api/endpoints/organization_metrics_estimation_stats.py

```diff
@@ -149,7 +149,10 @@ def estimate_volume(
 
 
 def _get_value(elm: MetricVolumeRow) -> float:
-    return cast(List[CountResult], elm[1])[0].get("count", 0.0)
+    ret_val = cast(List[CountResult], elm[1])[0].get("count", 0.0)
+    if ret_val is None:
```

Comment:
Since `None` is the default and you will replace the value, you could also do:
```suggestion
    ret_val = cast(List[CountResult], elm[1])[0].get("count")
    if ret_val is None:
```